examples/tutorial/tutorial_notebooks/introduction/exercise_3.ipynb

```diff
@@ -15,7 +15,7 @@
    "source": [
     "# Exercise 3: Not Implemented\n",
     "\n",
-    "**GOAL**: Learn what happens when a function is not yet supported in Modin and what functionality is not possible to accelerate"
+    "**GOAL**: Learn what happens when a function is not yet supported in Modin as well as how to extend Modin's functionality using the DataFrame Algebra."
```

Comment:
```suggestion
    "**GOAL**: Learn what happens when a function is not yet supported in Modin as well as how to extend Modin's functionality using the Dataframe Algebra."
```
@devin-petersohn, AFAIR, we agreed to use `Dataframe` instead of `DataFrame`.